src/App.tsx

```diff
@@ -1,35 +1,107 @@
+import { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Todo } from './types/Todo';
 
 export const App = () => {
+  const [title, setTitle] = useState('');
+  const [hasTitleError, setHasTitleError] = useState(false);
+
+  const [userId, setUserId] = useState(0);
+  const [hasUserIdError, setHasUserIdError] = useState(false);
+
+  const [todos, setTodos] = useState<Todo[]>(todosFromServer);
+
+  const getBiggestId = () => Math.max(...todos.map(todo => todo.id));
+
+  const getUserById = (): number => {
+    const user = usersFromServer.find(elem => elem.id === userId);
+
+    return user ? user.id : 0;
+  };
```

Comment:
The `getUserById` function is unnecessary in its current form. Since `userId` is already stored in the state, you can directly use `userId` instead of calling this function to get the same value. Consider removing this function and using `userId` directly in the `newTodo` object.